mewarpx/examples/capacitive_discharge.py

```diff
@@ -163,8 +170,10 @@ def _get_rho_ions():
 ##########################
 # simulation run
 ##########################
-
-mwxrun.simulation.step(max_steps - diag_steps)
+if args.steps:
```

Comment:
There is another call to simulation.step further down, that you will need to skip when running on circleCI.